apps/artemis-web/libs/csrf.ts

```diff
@@ -3,6 +3,7 @@ import { nextCsrf } from './next-csrf';
 
 const options = {
   secret: process.env.CSRF_SECRET, // Long, randomly-generated, unique, and unpredictable value
+  csrfSecret: process.env.CSRF_SECRET,
```

Comment:
why do we use this twice?

Reply:
to avoid the random generation that causes some bugs